src/particleContainer/adapter/vectorization/RealVecDouble.h

```diff
@@ -417,8 +417,10 @@ class RealVec<double> {
 				const __m128 recip_ps = _mm_rcp_ps(denom_ps);
 
 				const real_vec inv_unmasked = _mm256_cvtps_pd(recip_ps); //12bit
-			#elif VCP_VEC_WIDTH == VCP_VEC_W_512
+			#elif VCP_VEC_TYPE == VCP_VEC_KNL or VCP_VEC_TYPE == VCP_VEC_KNL
```

Comment:
Bitte hier genauer anschauen.

Reply:
is fine, only for avx256 one could neglect the last iteration (48 of 52 bits correct)

Reply:
replied below